bin/local-env/install-composer.sh

```diff
@@ -30,7 +30,7 @@ fi
 # Check if the current Composer version is up to date.
 if [ "$CI" != "true" ] && ! [[ "$(composer --version)" == "Composer version $COMPOSER_VERSION "* ]]; then
 	echo -en $(status_message "Updating Composer..." )
-	composer self-update
+	composer self-update $COMPOSER_VERSION
```

Comment:
I really would just do `composer self-update --1` to stay on the latest 1.x release.

Reply:
I tried this and it didn't work. 
<img width="1113" alt="Screenshot 2020-11-13 at 12 57 41" src="https://user-images.githubusercontent.com/237508/99074544-d7811e80-25af-11eb-9d44-c4f8144377b6.png">

Like all other dependencies loading from NPM / composer, we should control the version we use and only up versions once they are tested, otherwise this composer 2 problem may come up again.

Reply:
Oh, perhaps it was added in a more recent version. Can you update to `1.10.17` and try again?

Aside: Why pin the version  to `1.10.1` instead of the most recent version of that branch?

Reply:
We have to pin to a version here. 
`curl -Ls -w %{url_effective} -o /dev/null https://github.com/composer/composer/releases/latest | rev | cut -d '/' -f 1 | rev` this gets the latest release and compares it against current version. So 2.1 would trigger an update even if there is nothing new on 1.x branch. 

I say we hardcode and work on composer 2 support another ticket. 

Aside: I thought 1.10.1 was the latest release, I must have not checked 🤦

Reply:
I'm fine with hardcoding to `1.10.17` for now, just thought that with `composer self-update --1` it would be easier to stay on the latest version there. There wouldn't be any breaking changes in v1 anymore.